R/stat-density-2d.r

```diff
@@ -149,6 +153,8 @@ StatDensity2d <- ggproto("StatDensity2d", Stat,
     } else { # lines is the default
       contour_stat <- StatContour
```

Comment:
I think you need to make a new object since you are modifying a field of it below and it will affect everything inheriting from `StatContour` afterwards
```suggestion
      contour_stat <- ggproto(NULL, StatContour)
```

Reply:
Yes, correct. It didn't matter in the past but it matters now. I'll fix it.